src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -61,17 +61,19 @@ function sortAndFilterChartData(props) {
   return sortedCities;
 }
 
-function getSeries(props) {
-  const chartColors = props.chart.indicatorIds.length === 1
-    ? [getColorVariant(props.colorBase, 300)]
-    : getColorRange(props.colorBase);
+function getFill(chartColors, pos, props) {
+  return props.showChartPatterns ? `url(#${props.svgPatternBase}-${pos})` : chartColors[pos];
```

Comment:
If you didn't know... in SVG, the 'fill' attribute can be a colour or a URL pointing to the id of a definition elsewhere. In this case, some `<pattern>` defs.